src/components/metadataEditor/personEditor.template.html

```diff
@@ -12,33 +12,7 @@ <h3 class="formDialogHeaderTitle">
         </div>
 
         <div class="selectContainer">
-            <select is="emby-select" id="selectPersonType" class="selectPersonType" label="${LabelType}">
```

Comment:
This should not have been removed right?

Reply:
Isn't removed. <select is="emby-select" id="selectPersonType" class="selectPersonType" label="${LabelType}"></select> is still there.

Reply:
Hah whoops missed that last line. 😂